test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -829,11 +829,13 @@ def check_hardware_status_and_core_files(self):
                     self.logger.warning(_msg)
             for u in PBS_ALL_USERS:
                 user_home_files = du.listdir(hostname=hostname, path=u.home,
-                                             sudo=True, fullpath=False)
-                if fnmatch.filter(user_home_files, "core*"):
-                    _msg = hostname + ": user-" + str(u)
-                    _msg += ": core files found in "
-                    self.logger.warning(_msg + u.home)
+                                             sudo=True, fullpath=False,
+                                             runas=u.name)
+                if user_home_files is not None:
+                    if fnmatch.filter(user_home_files, "core*"):
```

Comment:
You can merge these two if you want
```suggestion
                if user_home_files and fnmatch.filter(user_home_files, "core*"):
```

Reply:
@vstumpf Thanks for suggestion. I have merged above two in present commit.